liota/core/package_manager.py

```diff
@@ -472,6 +486,10 @@ def run(self):
                     self.flag_alive = False
                     break
             else:
+                # currently, write 'Unsupported' to the named pipe once
+                #     successfully receives these commands
+                with open(self._rsp_pipe_file, "w+") as fp:
+                    fp.write('Unsupported\n')
```

Comment:
Same as before; since this is part of a client/server protocol we should have the strings somewhere global for both sides to include and stay consistent.

Reply:
done the same as above